src/libraries/Common/tests/System/Net/Sockets/TestSettings.cs

```diff
@@ -8,9 +8,10 @@ namespace System.Net.Sockets.Tests
     public static class TestSettings
     {
         // Timeout values in milliseconds.
-        public const int PassingTestTimeout = 10000;
```

Comment:
Can you keep this untouched to be consistent with other tests? e.g. `PassingTestTimeout` is normal/default timeout. If we need anything different for come case we would make it with specific name - like `PassingTestLongTimeout` (that part looks ok to me)

Reply:
Reverted this renaming to reduce the diff size.

I think more verbosity in the names wouldn't hurt here. 10 sec is quite a short time in comparison to Http.Tests's default `PassingTestTimeout=30sec`. We use hardcoded 30 seconds at a bunch of places in socket tests, which we will eventually replace with `PassingTestLongTimeout`, meaning that the "default" `PassingTestTimeout` will become much less common.